## Lab # 10 Microbiome and Analytics

## Table of Contents
1. [Introduction](#intro)
2. [Analytics Guest Speakers](#speakers)

<a name="intro"></a>
## Introduction

The goal of this lab is to review the analysis of microbiome 16S profiling data.

**Lectures** - [Microbiome](https://github.com/agmcarthur/Biochem-3BP3/blob/master/Lectures/Lecture%209%20-%20Microbiome.pptx)

**Background Reading** (optional)
* Santelli et al. 2008. Abundance and diversity of microbial life in ocean crust. [Nature 453: 653-6](https://www.ncbi.nlm.nih.gov/pubmed/?term=18509444)
* Stearns et al. 2015. Culture and molecular-based profiles show shifts in bacterial communities of the upper respiratory tract that occur with age. [ISME J. 9:1246-59](https://www.ncbi.nlm.nih.gov/pubmed/?term=25575312)

<a name="speakers"></a>
## Analytics Guest Speakers

**Flash Updates**
* *Random Forest*. Provide an overview of the Random Forest method for classification of complex data. see [An Introduction to Random Forest](https://towardsdatascience.com/random-forest-3a55c3aca46d),[Proc Natl Acad Sci U.S.A. 115:1690-1692](https://www.ncbi.nlm.nih.gov/pubmed/?term=29440440), and [Front Genet. 9:297](https://www.ncbi.nlm.nih.gov/pubmed/?term=30123241)
* *Logistic Regression*. Provide an overview of Logistic Regression, a predictive machine learning method. See [Introduction to Logistic Regression](https://towardsdatascience.com/introduction-to-logistic-regression-66248243c148) and [mSystems 4:e00211-19](https://www.ncbi.nlm.nih.gov/pubmed/?term=31387929).
* *Natural Language Processing*. Provide an overview of Natural Language Processing for turning text into data, see [Introduction to Natural Language Processing for Text](https://towardsdatascience.com/introduction-to-natural-language-processing-for-text-df845750fb63), [BMC Bioinformatics 9:193](https://www.ncbi.nlm.nih.gov/pubmed/?term=18410678), and [Genome Biol. 2:S10](https://www.ncbi.nlm.nih.gov/pubmed/?term=18834488).

**Speakers**
* Kara Tsang (PhD Student) – Using machine learning to predict phenotype from genotype.
* Bhavya Singh (MSc Student) – Bioinformatics of metagenomics assembly.
* Nick Waglechner (PhD Student) – Phylogenomic reconciliation for drug discovery.
* Michael Chong (PhD Student) – Genome Wide Association Studies (GWAS) for understanding disease.
